lib/swagger-initializer.js

```diff
@@ -35,7 +35,10 @@ function swaggerInitializer (opts) {
 
     if (logoData) {
       const img = document.querySelector('#swagger-ui > section > div.topbar > div > div > a > img')
-      img.src = logoData
+
+      if (img) {
+        img.src = logoData
+      }
     }
```

Comment:
```suggestion
    if (
      logoData &&
      resConfig.layout === "StandaloneLayout"
    ) {
      const img = document.querySelector('#swagger-ui > section > div.topbar > div > div > a > img')
      img.src = logoData
    }
```

Reply:
The issue with this approach is that afaik swagger supports custom layouts, which may or may not contain a logo. Not sure if this is something this package supports, but if it does, and if I specify a custom layout with a logo, the above change wouldn’t patch the logo src

Reply:
If this package indeed supports custom layouts, this would actually be a breaking change